src/VisualStudio/Core/Def/Implementation/Preview/FileChange.cs

```diff
@@ -79,13 +80,18 @@ private ChangeList ComputeChildren(TextDocument left, TextDocument right, Cancel
 
             diffService = diffService ?? diffSelector.DefaultTextDifferencingService;
 
-            var diff = ComputeDiffSpans(oldText, newText, diffService, cancellationToken);
-            if (!diff.Differences.Any())
+            var diff = ComputeDiffSpans(diffService, left, right, cancellationToken);
+            if (diff.Differences.Count == 0)
             {
-                // There are no changes. Just show the whole document.
-                return GetEntireDocumentAsSpanChange(left);
+                // There are no changes.
+                return ChangeList.Empty;
```

Comment:
@heejaechang What would happen if there were changes in one file and no changes in another file /  project change? Are you saying that we end up showing the above "No Changes" check box for the file that didn't change and a separate check box for file / project that did change?